packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -16,8 +16,8 @@ export const azureActiveDirectory = (
     logout: (options?) => client.logout(options),
     signup: async (options?) => await client.loginPopup(options),
     getToken: async (options?: any) => {
-      const renewIdTokenRequest = options || {
-        scopes: [process.env.AZURE_ACTIVE_DIRECTORY_CLIENT_ID],
+      const authRequest = options || {
```

Comment:
Hi @jeliasson I see that `getToken` is being executed from the `AuthProvider.tsx` without the option to pass any arguments to it, may I know how do I pass in the `options` argument for the `getToken` function from my Redwood app? Thank you 🙏

![image](https://user-images.githubusercontent.com/61768800/117560203-3ededd80-b0be-11eb-90a4-2cfdec576c10.png)